base/setup/usetup/usetup.c

```diff
@@ -4598,14 +4604,18 @@ ProgressCountdown(
                                       24,
                                       TRUE,
                                       FOREGROUND_RED | BACKGROUND_BLUE,
-                                      TimerValue,
+                                      0,
                                       NULL,
                                       MUIGetString(STRING_REBOOTPROGRESSBAR),
                                       ProgressTimeOutStringHandler);
 
+    TimerValue = TimeOut * ProgressBar->Width;
```

Comment:
Do you think that using here `max(1, ProgressBar->Width);` would be ok, or not (and if so, why)?

Reply:
Correct, let it be consistent.